src/cabal-details.js

```diff
@@ -662,6 +732,28 @@ class CabalDetails extends EventEmitter {
       })
     })
 
+    // notify when a user has archived a channel
+    this.registerListener(cabal.archives.events, 'archive', (channel, reason, key) => {
+      const user = this.users[key]
+      if (key !== this.user.key && (!user || (!user.isModerator() && !user.isAdmin()))) { return }
```

Comment:
would logging here be useful for auditing purposes? i would expect this case to happen when someone was trying to be cheeky

Reply:
since the archiving is subjective, it is more likely for someone to just want to archive a channel for themselves than trying to be cheeky, i would say :)